### PR TITLE
updated changelog

### PR DESCRIPTION
**Pull request by wjblanke**

### CODE REVIEW

**Review by jack60612 — DISMISSED** [review]
🎉

File: CHANGELOG.md [hunk 1]
```diff
@@ -6,6 +6,79 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
...
+- Thanks to @PastaPastaPasta for some compiler warning fixes in bls-signatures
+- Thanks to @random-zebra for fixing a bug in the bls-signature copy assignment operator
+- Thanks to @lourkeur for fixes in blspy related to pybind11 2.8+
+- Thanks to @nioos-ledger with a fix to the python implementation of bls-signatures
+
+### Known Issues:
+
+- When you are adding plots and you choose the option to “create a Plot NFT”, you will get an error message “Initial_target_state” and the plots will not get created
+  - Workaround: Create the Plot NFT first in the “Pool” tab, and then add your plots and choose the created plot NFT in the drop down.
+- If you are installing on a machine for the first time, when the GUI loads and you don’t have any pre-existing wallet keys, the GUI will flicker and load anything.
```
Comment (hoffmang9) [inline CHANGELOG.md:76]: missing `not`?

**Review by hoffmang9 — CHANGES_REQUESTED** [review]
Nit.

**Review by esaung — DISMISSED** [review]
looks good!

File: CHANGELOG.md (hunk 1 above)
Comment (wjblanke) [reply CHANGELOG.md]: fixed

**Review by hoffmang9 — COMMENTED** [review]
lgtm

File: CHANGELOG.md (hunk 1 above)
Comment (esaung) [reply CHANGELOG.md]: good catch!

**Pull request merged**